Source/WebCore/Modules/webcodecs/WebCodecsVideoFrame.cpp

```diff
@@ -151,10 +151,14 @@ ExceptionOr<Ref<WebCodecsVideoFrame>> WebCodecsVideoFrame::create(CanvasImageSou
         if (!canvas->width() || !canvas->height())
             return Exception { InvalidStateError,  "Input canvas has a bad size"_s };
 
+#if ENABLE(MEDIA_STREAM)
```

Comment:
Is this the right guard?

Reply:
I can add that to the patch

Reply:
https://github.com/WebKit/WebKit/pull/6401